workflow/Snakefile

```diff
@@ -32,6 +34,8 @@ config = make_paths_absolute(config, os.getcwd())
 config["results_dir"] = os.path.join(os.getcwd(), config["results_dir"])
 RESULTS_DIR = config["results_dir"]
 E2G_DIR_PATH = config["E2G_DIR_PATH"]
+MODEL_DIR = config["model_dir"]
```

Comment:
Where is this being used?

Reply:
this is used in the ENCODE_rE2G/utils.smk function to get the model directory (since you suggested storing the models for scE2G in this repo and the DNase models in the former). So it's needed before importing the utils.smk code.

Reply:
makes sense! Can we add a comment?
e.g # Used in utils.smk for getting correct model